ios/NetworkModule.swift

```diff
@@ -68,6 +87,35 @@ public class NetworkModule: NSObject, RCTBridgeModule {
         }
     }
 
+    internal func preprocessHttpRequestFromJS(
+        nativeId: NativeId,
+        type: String,
+        request: HttpRequest,
+        completionHandler: @escaping (
+            _ request: HttpRequest
+        ) -> Void
```

Comment:
A note here: the style of arguments is inconsistent compared to `preprocessHttpResponseFromJS`. I think we should align them, but I honestly prefer this way.
What do you think if I change `preprocessHttpResponseFromJS` to use named parameters in the main PR?

Reply:
If you like it this way, feel free to update the other PR